.github/workflows/open-tickets.yml

```diff
@@ -38,6 +38,31 @@ jobs:
     with:
       version_file: centreon-open-tickets/www/modules/centreon-open-tickets/conf.php
 
+  check-version-consistency:
+    runs-on: ubuntu-24.04
+    needs: [get-environment]
+    if: |
+      needs.get-environment.outputs.skip_workflow == 'false' &&
+      needs.get-environment.outputs.stability == 'testing'
+    env:
+      version: ${{ needs.get-environment.outputs.major_version }}.${{ needs.get-environment.outputs.minor_version }}
+      version_file: .version.centreon-open-tickets
+      configuration_file: centreon-open-tickets/www/modules/centreon-open-tickets/conf.php
```

Comment:
I think the widget version should be handled here as well => https://github.com/centreon/centreon/blob/283aede52b65ccabc1303cf8c1cb7e547044937f/centreon-open-tickets/widgets/open-tickets/configs.xml#L7